Branch-SDK/Branch-SDK/BNCSystemObserver.m

```diff
@@ -73,7 +73,7 @@ + (NSString *)getURIScheme {
 }
 
 + (NSString *)getAppVersion {
-    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleVersion"];
+    return [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
```

Comment:
Like these suggestions. Original usage for CFBundleShortVersionString is because an app update requires a higher version to be approved. Thoughts?